source/security/openid-external-identity-management/external-authentication-with-openid-identity-provider.rst

```diff
@@ -126,10 +126,11 @@ provider for instructions on configuring user claims.
 
 MinIO provides :ref:`built-in policies <minio-policy-built-in>` for basic access
 control. You can create new policies using the :mc:`mc admin policy` command, or
-by using the MinIO Console. MinIO does not support assigning :ref:`groups
-<minio-groups>` to an :abbr:`OIDC (OpenID Connect)` managed identity. Specify
-any and all policies to attach to the user as part of its :abbr:`JWT (JWT)`
-policy claim.
+by using the MinIO Console. 
+
+MinIO does not support assigning an identity managed by :abbr:`OIDC (OpenID Connect)` to a MinIO :ref:`group <minio-groups>`. 
```

Comment:
@donatello to confirm, but until multi-IDP support is ready IIRC you cannot even create groups in MinIO. as written (previously and now) might imply that you could still create a group in the first place.

I'd classify this as a nit as the behavior overall is still "true".